examples/zone-printer/README.md

```diff
@@ -0,0 +1,247 @@
+[Google Cloud Platform (GCP) zone](https://cloud.google.com/compute/docs/regions-zones/)
+where the application is running. 
+
+# How does it work?
+
+> **Note:** This section explains how the application is setup. Feel free to skip to the next section if you don't care about these
+details. 
+
+Zone printer is intended to be run in a Kubernetes cluster and is
+composed of: a ConfigMap (`app/zonefetch.yaml`), a Deployment
+(`app/nginx-dep.yaml`) and a Service (`app/nginx-svc.yaml`).
+
+The ConfigMap consists of a script that fetches GCP zone
+information from the
+[GCE metadata server](https://cloud.google.com/compute/docs/storing-retrieving-metadata). The Deployment consists of a pod
+spec with two containers. One container mounts the aforementioned
+ConfigMap, executes the script in the ConfigMap and writes the
+output to `/usr/share/nginx/html/index.html`. The other container
+runs a standard nginx server that reads from `/usr/share/nginx/html/`.
+
+The pod, therefore, exposes the fetched zone information at
+`/` or `/index.html` HTTP paths.
+
+The Service exposes the Deployment pods on port `30061` on all the
+nodes in its Kubernetes cluster.
+
+The example also includes an Ingress manifest (`ingress/nginx.yaml`)
+that will be used by `kubemci`.
+
+# Tutorial
+
+## 0. Prerequisite
+
+### 0.1 GCP Project
+
+Ensure you that you have a GCP Project. If you don't have one already,
+you can create a project by following the instructions at
+[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+After your project is created, run the following command in your
+terminal:
+
+```shell
+PROJECT=<your-project-name>
+```
+
+### 0.2 `gcloud` installed and initialized
+
+You can install `gcloud` by following the instructions at
+[Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
+initialize it by running the following commands:
+
+```shell
+gcloud init # Set $PROJECT as the default project
+gcloud auth login
+gcloud auth application-default login
+```
+
+### 0.3 kubectl
+
+You can install `kubectl` by running:
+
+```shell
+gcloud components install kubectl
+```
+
+### 0.4 Kubernetes clusters on GCP
+
+#### Existing clusters
+
+> **Note:**: If you don't already have Kubernetes cluster on GCP,
+skip to the next subsection on [creating new clusters](#creating-new-clusters).
+
+`kubemci` requires Kubernetes clusters that are v1.8.1 or newer. Please
+ensure that all the clusters you are using in this tutorial satisfy the
+minimum version requirement. You can check your cluster version using
+the `kubectl version` command.
+
+If you already have clusters with their credentials in your local
+kubeconfig (`$HOME/.kube/config` by default) file, you need to ensure
+that you either only have clusters you are going to use for this
+tutorial in your default kubeconfig file or create a new kubeconfig
+file that just contains credentials for those clusters. Assuming that
```

Comment:
We should say that we recommend creating a new kubeconfig.
Also we should call the kubeconfig in this tutorial zoneprinterkubeconfig rather than mcikubeconfig.
They should think of that kubeconfig as a special kubeconfig for their zone printer multi cluster load balancer. They should edit it only if they want to change the clusters that the zone printer load balancer is spread to.

Reply:
> We should say that we recommend creating a new kubeconfig.
> They should think of that kubeconfig as a special kubeconfig for their zone printer multi cluster load balancer. They should edit it only if they want to change the clusters that the zone printer load balancer is spread to.

`--kubecontexts` should obviate this need and recommendation right?